test/integration/galera_configuration/controls/galera_spec.rb

```diff
@@ -1,5 +1,10 @@
-include_dir = '/etc/mysql/conf.d'
-include_dir = '/etc/my.cnf.d' if os.family == 'redhat'
+if os.family == 'redhat'
```

Comment:
This can be shortened to `os.redhat?`

https://docs.chef.io/inspec/resources/os/#osfamily-helpers